src/main/java/com/idrsolutions/microservice/JPedalServlet.java

```diff
@@ -150,17 +148,17 @@ protected void convert(Individual individual, Map<String, String[]> params,
         }
     }
 
-    private static void validateSettings(final Map<String, String> paramMap, final Mode mode) throws Exception {
-        final SettingsValidator settingsValidator = new SettingsValidator(paramMap);
+    @Override
+    protected SettingsValidator validateSettings(final String[] conversionParams) {
+        final SettingsValidator settingsValidator = new SettingsValidator(conversionParams);
 
-        if (mode == Mode.convertToImages) {
-            settingsValidator.requireString("format", validEncoderFormats);
-            settingsValidator.optionalFloat("scaling", new float[]{0.1f, 10});
+        final String mode = settingsValidator.validateString("mode", new String[] {"convertToImages"}, true);
+        if (mode.equals("convertToImages")) {
```

Comment:
@KieranFrance thoughts on using the following so that code navigation works better?
```java
if (mode.equals(Mode.convertToImages.name())) {
```

Reply:
@leonatherton good points. Most recent commits on this branch include the suggested changes. Can you review and confirm?